dspace-api/src/main/java/org/dspace/eperson/Group.java

```diff
@@ -252,4 +254,24 @@ void setPermanent(boolean permanence)
         permanent = permanence;
         setModified();
     }
+
+    /**
+     * Syntactically check if a group was not created by DSpace.
+     * Was this group created by a user directly?
+     *
+     * @param g the group to be evaluated
+     * @return true if the group was not created by the system.
+     */
+    public static boolean isCustom(Group g){
+        String pattern = "^(COMMUNITY|COLLECTION)_[A-Fa-f\\-0-9]+_[A-Z_0-9]+";
```

Comment:
In the future, we might want to make this more robuts by defining and setting an additional group property when creating groups.